lib/ClimaCoreVTK/src/ClimaCoreVTK.jl

```diff
@@ -127,5 +127,37 @@ function writevtk(
     end
 end
 
+"""
+    writevtk(
+        basename::String,
+        times,
+        fields,
+        [ispace=vtk_space(first(first(fields)));
+        vtkargs...
+    )
+
+Write a sequence of NamedTuple fields `fields` at times `times` as a vtk collection.
+"""
+function writevtk(
+    basename::String,
+    times,
+    fields::NamedTuple,
+    ispace = vtk_space(first(first(fields)));
+    vtkargs...,
+)
+    npad = ndigits(length(times); pad = 3)
+
+    paraview_collection(basename) do pvd
```

Comment:
you need to create a paraview collection for each key value tuple pair in fields, so this should destructure the tuple and then pass the key value pairs to the previous defined function.